e2e-tests/tfm/cypress/e2e/journeys/utilisation-reports/pdc-reconcile/mark-report-as-done.spec.js

```diff
@@ -59,47 +55,40 @@ context(`${PDC_TEAMS.PDC_RECONCILE} users can mark reports as done and not done`
       cy.wrap(visibleBanks).its('length').should('be.gte', 3);
     });
 
-    const mockUtilisationReportDetailsWithoutId = [];
+    const mockUtilisationReportDetails = [];
     cy.wrap(visibleBanks).each((bank, index) => {
-      const reportBank = { id: bank.id, name: bank.name };
+      const bankId = bank.id;
+      const reportId = bankId;
       const status = statusWithBankId.at(index)?.status;
 
       if (!status || status === UTILISATION_REPORT_RECONCILIATION_STATUS.REPORT_NOT_RECEIVED) {
-        mockUtilisationReportDetailsWithoutId.push(
-          createNotReceivedReportDetails(reportBank, {
-            reportPeriod,
-          }),
-        );
+        const mockNotReceivedReportDetails = UtilisationReportEntity.createNotReceived({
```

Comment:
Coming back to your pain you mentioned of reportDetails being the same for both single and plural. Now that in SQL we just have UtilisationReportEntity could we just call these utilisationReports in code and variable names? Quite a few timees we refer to it so not sure if it's just too big a refactor for now/if we want it

Reply:
I have refactored it everywhere because why not 🚀